ZuluContent/Spells/Sixth/Dispel.cs

```diff
@@ -1,39 +1,79 @@
 using System.Threading.Tasks;
+using Scripts.Zulu.Engines.Classes;
 using Server.Items;
 using Server.Mobiles;
+using Server.Network;
 using Server.Targeting;
+using ZuluContent.Zulu.Engines.Magic;
 
 namespace Server.Spells.Sixth
 {
     public class DispelSpell : MagerySpell, ITargetableAsyncSpell<Mobile>
     {
-        public DispelSpell(Mobile caster, Item spellItem) : base(caster, spellItem) { }
-        
+        public DispelSpell(Mobile caster, Item spellItem) : base(caster, spellItem)
+        {
+        }
+
         public async Task OnTargetAsync(ITargetResponse<Mobile> response)
         {
+            var magery = Caster.Skills[SkillName.Magery].Value;
+            Caster.FireHook(h => h.OnModifyWithMagicEfficiency(Caster, ref magery));
+
             if (!response.HasValue)
+            {
+                if (response.InvalidTarget is IDispellable {Dispellable: true} item)
+                {
+                    Effects.SendLocationParticles(
+                        EffectItem.Create(item.Location, item.Map, EffectItem.DefaultDuration),
+                        0x3729, 9, 20, 5042);
+                    Effects.PlaySound(item.Location, Caster.Map, 0x201);
+
+                    if (!Caster.ShilCheckSkill(SkillName.MagicResist, (int) magery, 25))
+                        item.Delete();
+                    else
+                        Caster.SendMessage("You failed to dispell the field.");
```

Comment:
dispell should be dispel no?

Reply:
also this could be a sendfailuremessage :)

Reply:
Ah yeah nice spot!